components/skus/browser/rs/lib/Cargo.toml

```diff
@@ -12,19 +12,19 @@ path = "tests/e2e.rs"
 required-features = ["e2e_test"]
 
 [dependencies]
-http = { version = "0.1" }
+http = { version = "0.2" }
 async-trait = "0.1.64"
 rand = { version = "0.7" }
 serde_json = "1.0"
 sha2 = "0.9"
 data-encoding = "2.1.2"
-tracing = { version = "0.1", default-features = false }
+tracing = { version = "0.1", default_features = false, features = ["attributes"] }
 bigdecimal = { version = "0.1", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
 chrono = { version = "0.4", default-features = false, features = ["clock", "serde"] }
 uuid = { version = "0.8", features = ["serde"] }
 challenge-bypass-ristretto = { version = "1.0.1", features = ["serde_base64"] }
-futures-retry = "0.5.0"
+futures-retry = { git = "https://github.com/brave-intl/futures-retry", rev = "2aaaafb", version = "0.5.0" }
```

Comment:
AFAIK it should be the same since this is simply reverting this change so the WASM lib builds outside chromium again:
https://github.com/brave/brave-core/commit/b1c1343cd086f87ff3b9151660668f4b44ea0240#diff-7150adfa3b679ee58d52bb8ff9b958c1dfaf8259dd560b2c28b5b2390c5ed51eL27

Reply:
The vendored copy claims to be d3ba6a80578ebf5a2a996664902d18e048cd55f5. Maybe it's just README.chromium that's out of date?

Reply:
urgh, no you're almost certainly right.

I talked about this dep with @bridiver earlier, he mentioned that the chromium vendoring doesn't support git tags - hence why we have both `rev` and `version` here. if their tooling doesn't support `rev` - what's the preferred move here? should we manually update the vendored lib? or would we be okay with just living with the discrepancy?

Reply:
For a small dep like this, I'd just update manually, paying the cost until we can improve the tooling. Without automation, discrepancies will happen, but efforts we make to keep things aligned speed up future debugging and review work.

Another option is to either get the upstream maintainers to take our changes, or fork properly with our own packages on crates.io, which the chromium tooling is better at.